download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -2992,6 +2992,111 @@ uiScoreDirectorFactory = solverFactory.getScoreDirectorFactory();
 
 == From 7.37.0.Final to 7.38.0.Final
 
+[.upgrade-recipe-major]
+=== Annotation scanning has been deprecated for removal
+
+The `<scanAnnotatedClasses/>` directive in solver configuration has been deprecated.
+Our new Quarkus and Spring Boot quickstarts provide much better support for annotation processing.
```

Comment:
- Avoid "we" and "our" in docs.
- Not sure if that states that Quarkus and Spring Boot do or do not that too. Proposed change:

```
Use the link:../../compatbility/quarkus.html[Quarkus extension] or link:../../compatbility/springBoot.html[Spring Boot starter] to automatically scan for annotated classes instead.
```